WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsAlertsProvider.swift

```diff
@@ -45,6 +45,13 @@ protocol CardReaderSettingsAlertsProvider {
                           continueSearch: @escaping () -> Void,
                           cancelSearch: @escaping () -> Void)
 
+    /// Defines an alert indicating connecting failed because their address needs updating.
+    /// The user may try again or cancel
+    ///
+    func connectingFailedMissingAddress(from: UIViewController,
```

Comment:
Instead of MissingAddress, would it be more appropriate to say InvalidStoreAddress or BadStoreAddress?

Reply:
In light of #5560, which I noticed after I reviewed this PR, I'd opt for "Incomplete" instead of "Missing"